hunspell.yaml

```diff
@@ -49,11 +49,6 @@ subpackages:
       - uses: split/manpages
     description: hunspell manpages
 
-  - name: hunspell-lang
```

Comment:
Should we withdraw this package too?

Reply:
It was empty anyway, hunspell doesn't provide locales